src/components/Callout/style.postcss

```diff
@@ -0,0 +1,51 @@
+@import 'main.postcss';
+
+.Callout {
+  &_trigger {
+    cursor: pointer;
+    outline: 0;
+  }
+
+  &_notch,
+  &_popup {
+    position: absolute;
+  }
+
+  &_notch {
+    border-bottom: 10px solid var(--colorBorder);
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    z-index: var(--zIndex-8--notificationsDropdownNotch);
+
+    &::after {
+      border-bottom: 9px solid #fbfbfb;
```

Comment:
I thought he had finally aligned all the colours that the UI was gonna use. Have the designers added a new one now? 😔